components/scream/src/share/util/scream_common_physics_impl.hpp

```diff
@@ -0,0 +1,281 @@
+#ifndef COMMON_PHYSICS_IMPL_HPP
+#define COMMON_PHYSICS_IMPL_HPP
+
+#include "share/util/scream_common_physics_functions.hpp"
+#include "physics/share/physics_constants.hpp"
+
+namespace scream {
+  
+//-----------------------------------------------------------------------------------------------//
+// Applies Exners Function which follows:
```

Comment:
Why not put these comments in the header describing the interface? None of them are related to the implementation of the function itself